iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergedLine.java

```diff
@@ -557,4 +617,18 @@ public HalfLine getHalf(Side side) {
                 throw new AssertionError("Unknown branch side " + side);
         }
     }
+
+    public double getValueForLimit(Terminal t, LimitType type) {
+        switch (type) {
+            case ACTIVE_POWER:
+                return t.getP();
+            case APPARENT_POWER:
+                return Math.sqrt(t.getP() * t.getP() + t.getQ() * t.getQ());
+            case CURRENT:
+                return t.getI();
+            case VOLTAGE:
```

Comment:
Same proposal?

Reply:
As discussed voltage is not (yet?) supported as a LimitType.